lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -97,11 +97,11 @@ sub engine_workit($) {
 
     my $nd = $job->{settings}->{NUMDISKS} || 2;
     for my $i (1 .. $nd) {
-        my $hdd = $job->{settings}->{"HDD_$i"} || undef;
-        if ($hdd) {
-            $hdd = locate_asset('hdd', $hdd, 1);
+        my $hddname = $job->{settings}->{"HDD_$i"} || undef;
```

Comment:
this was introduced by @aplanas  in 231d226a809c8218d213af8b0fa9eaf92d31e97d - so I assume this is his perl variant of how you provide defaults to hash accesses in python :)